pkg/loop/internal/pb/relayer.proto

```diff
@@ -256,6 +256,56 @@ message FromAccountReply {
   string Account = 1;
 }
 
+service ChainReader {
+  rpc RegisterEventFilter (RegisterEventFilterRequest) returns (RegisterEventFilterReply) {}
+  rpc UnregisterEventFilter (UnregisterEventFilterRequest) returns (RegisterEventFilterReply) {}
+  rpc QueryEvents (QueryEventsRequest) returns (QueryEventsReply) {}
+  rpc GetLatestValue (GetLatestValueRequest) returns (GetLatestValueReply) {}
+}
+
+// TODO
+message RegisterEventFilterRequest {}
+
+// TODO
+message RegisterEventFilterReply {}
+
+// TODO
+message UnregisterEventFilterRequest {}
+
+// TODO
+message UnregisterEventFilterReply {}
+
+// TODO
+message QueryEventsRequest {}
+
+// TODO
+message QueryEventsReply {}
+
+message VersionedByteArray {
+    uint32 version = 1;
+    bytes data = 2;
+}
+
+// GetLatestValueRequest has arguments for [github.com/smartcontractkit/chainlink-relay/pkg/types.ChainReader.GetLatestValue].
+message GetLatestValueRequest {
+    BoundContract bc = 1;
+    string method = 2;
+    VersionedByteArray params = 3;
+    VersionedByteArray retVal = 4;
```

Comment:
We don't need `retVal` here on the `Request`, right?
```suggestion
```

Reply:
Good point